playwright/mocks/generateLibraries.ts

```diff
@@ -0,0 +1,86 @@
+import { SharedSlice } from "@prismicio/types-internal/lib/customtypes";
+
+import { generateRandomId } from "../utils";
+
+type GenerateTypesArgs = { nbSlices: number };
```

Comment:
What does `nbSlices` mean? "nb" is unclear - is there a clearer name?

Reply:
Yes sorry my bad here, `slicesCount` is better!